test/e2e/framework/metrics_util.go

```diff
@@ -206,21 +206,22 @@ func (l *PodStartupLatency) PrintJSON() string {
 	return PrettyPrintJSON(PodStartupLatencyToPerfData(l))
 }
 
-type SchedulingLatency struct {
-	Scheduling LatencyMetric `json:"scheduling"`
-	Binding    LatencyMetric `json:"binding"`
-	Total      LatencyMetric `json:"total"`
+type SchedulingMetrics struct {
+	SchedulingLatency LatencyMetric `json:"schedulingLatency"`
```

Comment:
schedulingLatency -> scheduling_latency will be better in json.
and others like this.

Reply:
I don't agree. In the whole Kubernetes API we use camelCase for json tag - let's not come up with a different convention here.

Reply:
OK, make sense.